apps/s_cb.c

```diff
@@ -819,7 +819,8 @@ int generate_stateless_cookie_callback(SSL *ssl, unsigned char *cookie,
 {
     unsigned int temp;
     int res = generate_cookie_callback(ssl, cookie, &temp);
-    *cookie_len = temp;
+    if (res != 0)
```

Comment:
Add the blank line above while you're here?

Reply:
sure.